library/README.md

```diff
@@ -9,8 +9,7 @@ The version and status of each is summarized below:
 
 directory       | project         | version   | status
 ----------------|-----------------|-----------|---------------
-rssbuilder      | RSSBuilder      | 2.2.1     | patched [1]
-utf8            | phputf8         | 0.5       | unpatched
+rssbuilder      | RSSBuilder      | 2.2.1     | patched [2]
```

Comment:
Should this be `patched [1]`?

Reply:
It should.
I fixed this some while ago
https://github.com/mantisbt/mantisbt/commit/e30f40ce74d414b9604461eae3de88be29cc539d
but got lost when rebasing this branch.
Will fix.